src/plugins/gui/scenes/AddressFormScene.tsx

```diff
@@ -104,11 +254,45 @@ export const AddressFormScene = React.memo((props: Props) => {
   )
 })
 
-const getStyles = cacheStyles((theme: Theme) => ({
-  formSectionTitle: {
-    marginLeft: theme.rem(0.5),
-    marginTop: theme.rem(1),
-    marginBottom: theme.rem(0.5),
-    fontFamily: theme.fontFaceBold
+const getStyles = cacheStyles((theme: Theme) => {
+  const dropContainerCommon: ViewStyle = {
+    backgroundColor: theme.modal,
+    borderBottomLeftRadius: theme.rem(1),
+    borderBottomRightRadius: theme.rem(1),
+    zIndex: 1,
+    borderColor: theme.iconTappable,
+    borderTopWidth: 0,
+    borderWidth: theme.thinLineWidth,
+    overflow: 'hidden',
+    position: 'absolute',
+    left: theme.rem(1),
+    right: theme.rem(1)
+  }
+  return {
+    addressHintText: {
+      marginHorizontal: theme.rem(0.5),
+      marginVertical: theme.rem(0.25)
+    },
+    dropContainer: {
+      top: theme.rem(9.75),
```

Comment:
Move the top down 0.5 rem so it doesn't bump up against the text input field. It looks weird based on the screenshot.